editcompletion.php

```diff
@@ -70,6 +79,22 @@
     $date = time();
 }
 
+//function to reset completion for $users
+if (isset($_POST['reset_completion'])) {
```

Comment:
can you please change this one to an optional_param call? - direct access to $_POST is usually not allowed.

Reply:
it should also have a "confirm_sesskey()" check on that line as well.